src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2441,14 +2441,14 @@ static void sub_6CBCE2(
         // Set the temporary track element
         _tempTrackTileElement.SetType(TILE_ELEMENT_TYPE_TRACK);
         _tempTrackTileElement.SetDirection(trackDirection);
-        _tempTrackTileElement.AsTrack()->SetHasChain((edx & 0x10000) ? true : false);
+        _tempTrackTileElement.AsTrack()->SetHasChain((edx & 0x10000) != 0);
```

Comment:
I find the old notation easier to read.

Reply:
I'm inclined to disagree. This sort of boolean expression does not warrant a ternary operator, imo.

Reply:
I'm not very fussed, really. Let's keep @ShFil119 's way, then.